src/Compilers/VisualBasic/Portable/Lowering/ExpressionLambdaRewriter/ExpressionLambdaRewriter.vb

```diff
@@ -149,7 +149,7 @@ lSelect:
                     If expression IsNot Nothing Then
                         Return Visit(expression)
                     End If
-                    ' Otherwise fall through and generate an error
+                ' Otherwise fall through and generate an error
```

Comment:
Preserve the original indentation?
Also, it looks like the comment is misleading, we are exiting Select Case here rather than falling through the next Case.